asq-java-q.html

```diff
@@ -8,8 +9,10 @@
 
 Example:
 
-    <asq-java-q tabsize="2">
-      <asq-stem><h4>Implement a for loop in Java</h4></asq-stem>
+    <asq-java-q exercise-name="exercise3" tabsize="2" compile-timeout-ms="2000" execution-timeout-ms="1" >
```

Comment:
add docs for the important properties  (non-private) and methods.

Example: https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L36